src/components/icons/App.svelte

```diff
@@ -23,6 +23,7 @@
 <svg
   on:click
```

Comment:
Do we need any of these event handlers for the tool/climate icons? I think the only place they were formerly interactive was on the home page. If they're to be used interactively in the future then wrapping them in a `<button>` element would be a more appropriate way to handle these event listeners.

Reply:
In fact, we should probably remove most of the aria related & title props. The `<svg>` element should have an `alt` attribute that is an empty string (`""`) if no alt text is given or a description of what the icon's purpose is otherwise (e.g. `alt= "Filter climate tools by sun category"`). 

Using `aria-labelledby` in combination with the SVG `<title>` element is a better alternative to using `alt` text for inlined SVG, however because all of these icon components use the same `id` for their title, that will mess things up. 

To fix the a11y my recommendation would be to 

1. add a unique id for each title, e.g. `icon-title-rainfall`
2. use `aria-labelledby="unique-icon-title-id"` on the SVG element
3. add `role="img"` on the SVG element

The contents of the SVG title element could be dynamic via a prop on the component with the fallback being "<Icon Name> Icon".

Reply:
This is a good article on accessible SVG btw: https://css-tricks.com/accessible-svgs

Reply:
Thanks for the recommendations! Let me know if the fixes in https://github.com/berkeley-gif/caladapt-website-2021/pull/166/commits/8c97c84274795025eeb80d24c5d57174952a4a21 are correct.

Reply:
No problem. I left a couple comments on the App Icon file.

Reply:
@mukhtyar I'm realizing now the a11y fix I mentioned previously is only useful for when the SVG is informative for a user. However, we are only using these icons in a decorative manner in the UI. AFAIK no where are they clickable, meaning we're not using them wrapped in a button with no visual text. For that reason I think it would make sense to either: 

1. Have the title text be a prop that defaults to `undefined`. If the value is truthy, then we would render the `<title>` within the SVG with the title text as well as add the `aria-labelled-by="title-id"` and `role="img"`. If the value of the title text prop is falsey, then we would add `role="presentation"` and `aria-hidden=true` to the SVG element as well as not render the `<title>`.

2. Strip out all of the following SVG id, aria-labelled-by, `<title>` and hardcode the `role="presentation"` and `aria-hidden=true` attributes to the SVG making the icons only decorative.

[Here's an article that explains more](https://scottvinkle.me/blogs/work/svg-and-emoji-icon-accessibility)

I apologize for the back and forth on this and I'm happy to make the changes if you'd prefer not to.

Reply:
@clhenrick No worries. I can make the updates. You have enough on your plate with the SLR tool.